### PR TITLE
Migrate styling from Bootstrap to MUI v5

### PR DESCRIPTION
**Pull request by DivyanshuParwal**
#11 resolved. Replaced Bootstrap components with temporary equivalent MUI components. Removed Bootstrap completely including all related packages and imports. Also removed Nav.module.css file as no longer required.

### CODE REVIEW

File: src/pages/about.jsx [hunk 1]
```diff
@@ -36,90 +38,30 @@ export const About = () => {
...
-            <Card.Img
-              variant="top"
-              src="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
-            />
-            <Card.Body>
-              <Card.Title>Salish Sea Vessel Research</Card.Title>
-            </Card.Body>
-          </Card>
-        </CardColumns>
+        <Card sx={{ maxWidth: 345, marginBottom: '20px' }}>
```
Comment (paulcretu) [inline src/pages/about.jsx:41]:
```suggestion
        <Card sx={{ maxWidth: 345, mb: 2 }}>
```

Ideally use the [MUI spacing](https://mui.com/system/spacing/) utility when possible. Default unit is 8px so `mb: 2` = 16px

File: src/pages/about.jsx [hunk 2]
```diff
@@ -36,90 +38,30 @@ export const About = () => {
...
+          <CardMedia
+            component="img"
+            height="140"
+            image="https://www.aquariumcostadealmeria.com/wp-content/uploads/2019/01/shutterstock_554899423-1-1280x720.jpg"
+          />
+          <CardContent>
+            <Typography gutterBottom variant="h5" component="div">
```
Comment (paulcretu) [inline src/pages/about.jsx:48]:
```suggestion
            <Typography gutterBottom variant="h5">
```

Does it need to be a div? What's wrong with `h5`?

File: src/pages/getinvolved.jsx [hunk 3]
```diff
@@ -87,42 +87,13 @@ export const getinvolved = () => {
...
-                  , especially for the Souther Resident Killer Whales
-                </p>
-                <footer className="blockquote-footer">
-                  <small className="text-muted">
-                    Listen Live for Orcas in{' '}
-                    <cite title="Source Title">Source Title</cite>
-                  </small>
-                </footer>
-              </blockquote>
-            </Card>
```
Comment (paulcretu) [inline src/pages/getinvolved.jsx:118]: Looks like you're missing this card in the new version

File: src/pages/about.jsx [hunk 4]
```diff
@@ -36,90 +38,30 @@ export const About = () => {
...
-            />
-            <Card.Body>
-              <Card.Title>Salish Sea Vessel Research</Card.Title>
-            </Card.Body>
-          </Card>
-        </CardColumns>
+        <Card sx={{ maxWidth: 345, marginBottom: '20px' }}>
+          <CardMedia
+            component="img"
+            height="140"
```
Comment (paulcretu) [inline src/pages/about.jsx:44]: If you're going to constantly be re-using a card with the same props (`component` and `height`) maybe it's worth making it into a new component or use `styled()`

**Review by paulcretu — CHANGES_REQUESTED** [review]
Thanks for doing the refactor! Just a few comments

**Comment by DivyanshuParwal** [comment]
Thanks @paulcretu for your comments. What I was thinking when working on this issue was to just remove bootstrap components with temp components which will ultimately be removed by the components according to the designs. So should I work on suggestions like creating a new reusable card component and creating a card for the missed one which I think is an unnecessary work ?

File: package.json [hunk 5]
```diff
@@ -18,14 +18,13 @@
...
     "@mui/material": "^5.3.1",
-    "bootstrap": "^4.6.0",
     "next": "^12.0.9",
     "react": "17.0.2",
     "react-audio-player": "^0.17.0",
-    "react-bootstrap": "^1.6.1",
     "react-dom": "17.0.2"
   },
   "devDependencies": {
+    "@types/node": "^17.0.13",
```
[the remark this answers is not in the document]
Comment (DivyanshuParwal) [reply package.json:27]: I’ll remove this

**Comment by paulcretu** [comment]
@DivyanshuParwal yeah that's a good point. I actually didn't mean to request changes for all the suggestions, just for removing `@types/node`. My mistake, I should have been more clear. All the other comments are feedback in case you find it helpful—they're not blocking. So go ahead and remove `package.json`/`package-lock.json` and I'll merge

**Comment by DivyanshuParwal** [comment]
@paulcretu I have removed @types/node

**Comment by paulcretu** [comment]
@DivyanshuParwal `package-lock.json` is still conflicting, you may want to rebase/merge changes from `main` into your branch

**Comment by DivyanshuParwal** [comment]
@paulcretu merge conflict resolved.

**Review by paulcretu — APPROVED** [review]

**Pull request merged**